types/node-postal/index.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for node-postal 1.1
+// Project: https://github.com/openvenues/node-postal
+// Definitions by: João Carmo <https://github.com/joaocarmo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Source: https://github.com/openvenues/libpostal#parser-labels
+export type PostalLabels =
+    | 'category'
+    | 'city_district'
+    | 'city'
+    | 'country_region'
+    | 'country'
+    | 'entrance'
+    | 'house_number'
+    | 'house'
+    | 'island'
+    | 'level'
+    | 'near'
+    | 'po_box'
+    | 'postcode'
+    | 'road'
+    | 'staircase'
+    | 'state_district'
+    | 'state'
+    | 'suburb'
+    | 'unit'
+    | 'world_region';
+
+export interface PostalResult {
+    component: PostalLabels;
+    value: string;
+}
+
+export function expand_address(address: string): string[];
+
+export function parse_address(address: string): PostalResult[];
+
+export const expand: {
+    expand_address: typeof expand_address;
+};
+
+export const parser: {
+    parse_address: typeof parse_address;
+};
+
+export const postal: {
+    expand: typeof expand;
+    parser: typeof parser;
+};
+
+export as namespace postal;
```

Comment:
```suggestion

export as namespace postal;
```
this is when module can be used in global scripts via symbol export, say within browser environment,
not really applicable here I think

Reply:
Ah! That makes sense. Thank you for pointing that out!